x-pack/test_serverless/functional/test_suites/common/management/ingest_pipelines.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { IngestPutPipelineRequest } from '@elastic/elasticsearch/lib/api/types';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const TEST_PIPELINE_NAME = 'test_pipeline';
+
+const PIPELINE = {
+  name: TEST_PIPELINE_NAME,
+  description: 'My pipeline description.',
+  version: 1,
+};
+
+const PIPELINE_CSV = {
+  name: TEST_PIPELINE_NAME,
+};
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['svlCommonPage', 'common', 'ingestPipelines']);
+  const es = getService('es');
+  const security = getService('security');
+  const log = getService('log');
+
+  describe('Ingest Pipelines', function () {
+    this.tags('smoke');
+    before(async () => {
+      await security.testUser.setRoles(['ingest_pipelines_user']);
```

Comment:
This will not work when we run on MKI - the serverless security model doesn't allow this kind of custom role. Right now, `pageObjects.svlCommonPage.login()` will login a superuser. You can either stay with that or, if you need role specific testing here, use the new serverless role testing capabilities that have recently been added. Here's an example of how to use that: https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/common/platform_security/viewer_role_login.ts

Reply:
thanks a lot for the review, @pheyos! I was not aware of this. We also have the same pattern in the index management tests, should I update them as well? 
This might be the reason for enrich policies tests to fail on MKI, @sabarasaba?

Reply:
@pheyos Removed the custom role in https://github.com/elastic/kibana/pull/172480/commits/4b522f19f0d6050d625dd06d66939a1bd8f2f622

Reply:
> We also have the same pattern in the index management tests, should I update them as well?

@yuliacech Yes, that makes sense (probably as a separate PR). This pattern doesn't work on MKI and thus should not be used in serverless tests.

Reply:
Thanks a lot for the explanation @pheyos! I'll open a follow up PR for index management tests